packages/esm-home-app/src/openmrs-esm-home-schema.ts

```diff
@@ -35,4 +35,9 @@ export const esmHomeSchema = {
       _validators: [validators.isUrlWithTemplateParameters(['patientUuid'])],
     },
   },
+  appNameLabel: {
```

Comment:
What's the use-case for having a configurable translation key here instead of just using the translation system directly?

Reply:
This was in response to [this comment](https://github.com/openmrs/openmrs-esm-home/pull/186#discussion_r1507211032)

Reply:
Could you maybe suggest a solution for this kindly?

Reply:
Sorry I didn't respond to this sooner, but the approach you've used in the most recent commits where it's just a string that gets passed in as a prop to `PageHeader` is appropriate. Implementers can change the label to whatever they want using [translation overrides](https://o3-docs.openmrs.org/docs/configure-o3/configure-translations.en-US#translation-overrides-specific-to-an-individual-o3-module).